code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
 	var/is_genetic = TRUE	// If TRUE, the marking is considered genetic and is embedded into DNA.
 	var/is_painted = FALSE	// If TRUE, the marking can be put on prosthetics/robolimbs.
 
-	tiger_stripes
-		name = "Tiger Stripes (Tajara)"
-		icon_state = "tiger"
-		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","chest","groin")
-		species_allowed = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
+	aug_headplug
+		name = "Augment (Headplug, Back Head)"
+		icon_state = "aug_headplug"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
 
-	taj_paw_socks
-		name = "Socks Coloration (Tajara)"
-		icon_state = "taj_pawsocks"
-		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand")
-		species_allowed = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
+		diode
+			name = "Augment (Headplug Diode, Back Head)"
+			icon_state = "aug_headplugdiode"
 
-	una_paw_socks
-		name = "Socks Coloration (Unathi)"
-		icon_state = "una_pawsocks"
-		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand")
-		species_allowed = list("Unathi")
+	aug_backports
+		name = "Augment (Backports, Back)"
+		icon_state = "aug_backports"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
 
-	belly_hands_feet
-		name = "Hands,Feet,Belly Color (Minor)"
-		icon_state = "bellyhandsfeetsmall"
-		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand","groin","chest")
-		species_allowed = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
+		diode
+			name = "Augment (Backports Diode, Back)"
+			icon_state = "aug_backportsdiode"
 
-	patches
-		name = "Color Patches"
-		icon_state = "patches"
-		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand","chest","groin")
-		species_allowed = list("Tajara", "Zhan-Khazan Tajara", "M'sai Tajara")
+	aug_backportswide
+		name = "Augment (Backports Wide, Back)"
+		icon_state = "aug_backportswide"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
```

Comment:
i dunno about tajara/base unathi having augments

Reply:
I DID consider providing an alternative name whenever it checks for the species so that it suggests something else, but I'm not sure what our doctrine is on extra code for the same icon.

Reply:
Would that be suitable or?